File: scode/TeamDigital/prob19/main.py
# a list of all the suits 
Suits = ['CLUB', 'DIAMOND', 
		'Heart', 'SPADE'] 
# a list of all the ranks 
Ranks = ['A', '2', '3', '4', 
		'5', '6', '7', '8', 
		'9', '10', 'J', 'Q', 'K'] 

# Matching all the suits with all the ranks 
for rank in Ranks: 

	for suit in Suits: 
		print(f'{rank} of {suit}'.ljust(16), end='') 

	# Spacer 
	print() 

File: scode/TeamDigital/prob19/main.c
#include<stdio.h>

int main(){
    printf("Bonjour TeamDigital !\nAllez, c'est à vous ! Bon courage ! ^_^");
    return 0;
} 
typedef enum suit 
{
	Club, Spade, Heart, Diamond
} Suit;

typedef enum rank 
{
	Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten, Jack, Queen, King, Ace 
} Rank ;

struct card
{
Suit c_suit;
Rank c_rank;
} Card;
card Deck[72];
